po/.gitignore

```diff
@@ -1,2 +1,3 @@
 *.mo
 *.po.new
+.downloaded
```

Comment:
It's now `.po-downloaded` and one level up.

Reply:
Fixed.